common/entryCreditBlock/commitchain.go

```diff
@@ -17,18 +17,21 @@ import (
 
 const (
 	// CommitChainSize = 1+6+32+32+32+1+32+64
+	// These are the sizes of the members of the data structure below
 	CommitChainSize int = 200
 )
 
+// CommitChain is a data structure which affects EC balances when committing a new user chain into the Factom blockchain
+// see https://github.com/FactomProject/FactomDocs/blob/master/factomDataStructureDetails.md#entry_commit
 type CommitChain struct {
-	Version     uint8                   `json:"version"`
-	MilliTime   *primitives.ByteSlice6  `json:"millitime"`
-	ChainIDHash interfaces.IHash        `json:"chainidhash"`
-	Weld        interfaces.IHash        `json:"weld"`
-	EntryHash   interfaces.IHash        `json:"entryhash"`
-	Credits     uint8                   `json:"credits"`
-	ECPubKey    *primitives.ByteSlice32 `json:"ecpubkey"`
-	Sig         *primitives.ByteSlice64 `json:"sig"`
+	Version     uint8                   `json:"version"`     // The version of the CommitChain, currently 0
+	MilliTime   *primitives.ByteSlice6  `json:"millitime"`   // The millisecond time stamp (0~=1970) this commit is created
```

Comment:
"0" is known as "unix epoch", so the expected comment here is (imo):
`// milliseconds since epoch as unsigned six-byte big endian integer`